menu.h

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include "utils.h"
+
+using namespace std;
```

Comment:
Delete the "using" line. Let's have explicit calls for the functions from the std lib.